packages/relay/src/lib/eth.ts

```diff
@@ -1489,7 +1489,11 @@ export class EthImpl implements Eth {
   ): Promise<string | JsonRpcError> {
     this.logger.error(
       e,
-      `${requestDetails.formattedRequestId} Failed to successfully submit sendRawTransaction for transaction ${transaction}`,
+      `${
+        requestDetails.formattedRequestId
+      } Failed to successfully submit sendRawTransaction for transaction ${transaction} to node ${JSON.stringify(
+        this.hapiService.getMainClientInstance().network,
```

Comment:
I don't think this outputs what you'd expect.
Do you have an example of the output you could put in the PR details for reviewers?
Each transaction could go to any number of noes in the network depending on the SDK details.

Reply:
This produces the following which may include information we do not need:
```
to node {"127.0.0.1:50211":{"shard":{"low":0,"high":0,"unsigned":false},"realm":{"low":0,"high":0,"unsigned":false},"num":{"low\":3,"high":0,"unsigned":false},"aliasKey":null,"evmAddress":null,"_checksum":null}}
```
Updated the `hapiServer.ts` to make the `hederaNetwork` public and use that in the logs which just lists:
```
to node {\"127.0.0.1:50211\":\"0.0.3\"}
```
The description has been updated

Reply:
That's an improvement but it's still not what you need.
network is the network the SDK is set to point to . In this case it's a local network so all transactions will go there.
if you were to have more than one node this would likely contain multiple nodes from what transaction would be sent to.

What we need is the actual node a transaction is sent to so we can say Transaction 1 was sent to Node 2 and Transaction 3 was sent to Node 4.

Try this against previewnet or testnet.

Reply:
Okay, so yes, the node to send a transaction to is selected by the internal round-robin process.  It looks as if the `nodeId` is available in the Hedera transaction response, however the problem we're trying to solve is identifying or capturing the `nodeId` on failed transactions to identify a potentially problematic node.  The message we're seeing in the logs is:
` Transaction execution returns a null value: transactionId=...`
That message comes from a `null` or `undefined` transactionResponse, and so the `nodeId` will not be available to log.
```
      if (!transactionResponse) {
        throw predefined.INTERNAL_ERROR(
          `${requestDetails.formattedRequestId} Transaction execution returns a null value: transactionId=${transaction.transactionId}, callerName=${callerName}, txConstructorName=${txConstructorName}`,
        );
      }
```